client/settings/payment-settings/index.js

```diff
@@ -34,7 +34,7 @@ const GeneralSettingsDescription = () => (
 			</ExternalLink>
 		</p>
 		<p>
-			<ExternalLink href="https://woocommerce.com/contact-us/">
+			<ExternalLink href="https://woocommerce.com/my-account/create-a-ticket?select=18627">
```

Comment:
This actually redirects me to: `https://woocommerce.com/my-account/contact-support/?select=18627`, should we use that link directly?

Reply:
Nice spot! Fixed in fa486a083c97b568dcf7450e8448fcf636a24461